### PR TITLE
Workaround for CircleCI build test

### PR DESCRIPTION
**Pull request by paescuj**
The build test over CircleCI doesn't work at the moment because the package libwebkit2gtk-4.0-dev isn't available in Ubuntu 14.04.

Since Ubuntu 16.04 won't be available at CircleCI in the near future and the compilation of WebkitGTK+ isnt' really an option in my opinion (complex and slow), I suggest adjusting the CircleCI configuration to simply use the xenial (Ubuntu 16.04) repository to get the mentioned package (tested on a Ubuntu 14.04 machine).

Though this is an ugly solution, we would at least have a working build test for the moment.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -3,15 +3,19 @@ machine:
 
 dependencies:
   post:
+    - sudo apt-get update
     - >
       sudo apt-get install -y
+      upstart
```
Comment (lots0logs) [inline circle.yml]: Why is upstart needed?

**Comment by lots0logs** [comment]
Thanks! I knew about the problem I just havent had the time to decide what to do about it :dizzy_face:

File: circle.yml (hunk 1 above)
Comment (paescuj) [inline circle.yml]: Please see https://github.com/Antergos/lightdm-webkit2-greeter/pull/74/commits/f1079f8a0aee0f9a9d26df218ec71daa143c4d2b for a detailed explanation. 

Also note that I've reverted this change now in https://github.com/Antergos/lightdm-webkit2-greeter/pull/74/commits/352460fd99a93463eb886bfcf2d984e5a2c0c5f4. At first I thought I could fix the mentioned error by upgrading upstart...

**Comment by paescuj** [comment]
> Thanks! I knew about the problem I just havent had the time to decide what to do about it :dizzy_face:

You're welcome :smiley: That's just what I was thinking - so I tried to help you out with this.
I'm glad, it finally works!

Have you already seen the [status badges of CircleCI](https://circleci.com/docs/status-badges/)?
Maybe you would like to add one into the README.

**Pull request merged**